### DISCUSSION TITLE
Vue Datatable component with Tailwind

### DISCUSSION THREAD

**Discussion by nla-sean** [primefaces/community#713] [PrimeVue] [answered]
Hi there,

Wondering when the tailwind version of datatable will be coming?

**Comment by tugcekucukoglu** [comment, marked as answer]
The team continues to work on this. DataTable is one of the most priority components. It will be completed soon.

Thanks for your interest!

**Reply by nla-sean** [reply]
Thanks @tugcekucukoglu this is awesome, the team are doing a wicked job :)
